### ISSUE TITLE
Feature request: migrate images in issues

### ISSUE THREAD

**Issue by archon810** [piceaTech/node-gitlab-2-github#69]
Currently, images are not migrated and result in 404s.

**Comment by spruce** [comment]
A MR ist verry welcome. 😄

**Comment by virzak** [comment]
#70 allows uploading attachments to S3.

**Comment by ajRiverav** [comment]
did #70 implement this?

**Comment by virzak** [comment]
@ajRiverav yes. The instructions are [here](https://github.com/piceaTech/node-gitlab-2-github/blob/master/README.md#s3-optional).

**Comment by ajRiverav** [comment]
ok. I wasn't sure since the issue is still open.

**Comment by ashkansk** [comment]
Uploading attachments into the S3 is a great feature, but I was wondering if it's possible to migrate issue images into the GitHub itself, instead of S3?

**Comment by virzak** [comment]
@ashkansk it cannot be done. See the first comment in #70

**Comment by mschoettle** [comment]
While there is no API, there is a way to do this. The project [`github-s3`](https://github.com/j178/github-s3) has replicated by using the unpublicized API.

I tested this out. Basically, instead of using S3, the attachment is downloaded into a temporary location, and then `github-s3` is called to upload the file to GitHub. The resulting URL is then used in the body.

Here is my diff: https://github.com/mschoettle/node-gitlab-2-github/commit/b90b3396c440b9306d24720aedf9f03457505e0f

I don't think that this can be officially used by this tool since it is using an unofficial way.
